src/details/ArborX_DetailsDistributor.hpp

```diff
@@ -108,7 +108,7 @@ determineBufferLayout(InputView batched_ranks, InputView batched_offsets,
   offsets.reserve(n_unique_ranks + 1);
   counts.reserve(n_unique_ranks);
 
-  for (unsigned int i = 0; i < n_unique_ranks; ++i)
+  for (int i = 0; i < n_unique_ranks; ++i)
```

Comment:
Isn't this totally unrelated? Shouldn't this be in a separate pull request?

Reply:
The testing was [failing](https://cloud.cees.ornl.gov/jenkins-ci/job/ArborX/job/PR-166/12/clang-tidy/new/) because style checks are part of the testing on Jenkins.

Reply:
Sure, it's all just a question of the philosophy of using `git`.